pgcli/pgexecute.py

```diff
@@ -307,6 +308,12 @@ def run(self, statement, pgspecial=None, exception_formatter=None,
                 if not on_error_resume:
                     break
 
+    def _cannot_catch(self, e):
+        # Raise operational errors regardless of on_error
+        # specification.
```

Comment:
Also, the comment says "Raise operational errors regardless of on_error specification", but it does not fully apply anymore, since we now raise selectively, and not on any OperationalError. Now that this logic moved into a method, there should also be a properly formatted docstring.